### PR TITLE
fixes #923 update cookbook examples and fix some typo

### LINKED ISSUE #923: [Documentation]: Some cookbook examples are out-of-date.
**Describe the issue**
Hi, I found some cookbook examples are out-of-date.

After we changed line `pkg/api/member.go:76` from `buff, err := yaml.Marshal(resp)` to `buff, err := codectool.MarshalJSON(resp)`, some `grep` commands in examples are not worked.

e.g. commands in [Getting Started](https://github.com/megaease/easegress#getting-started) and [Easegress cluster](https://github.com/megaease/easegress/blob/main/doc/cookbook/multi-node-cluster.md) cookbook:
```shell
$ egctl member list | grep "cluster-role"   # should grep "ClusterRole"
$ egctl member list | grep "api-addr"
$ egctl member list | grep "name"
```
```shell
egctl --server $HOST1:2381 member list | grep " name"
```

**Version**
Latest master branch version `73d7dd32`.

**Additional context**
I'm sure that everyone can fix this little tiny issue by themselves. But I also want to make our docs better, at least the 
`Getting Started` part.

---
Thanks for contributing 🎉!

### PR DESCRIPTION
**Pull request by grootpiano**
fixes #923

I noticed that we already opened a PR #882 for updating docs.
So, which branch should I merge these changes into, `megaease:main` or `localvar:minor-doc-fix`?

### CODE REVIEW

**Comment by grootpiano** [comment]
> please use `megaease:main`, or your contribution will belong to me 😄, because `localvar:minor-doc-fix` is my PR .

That doesn't matter. We all Groot. 👀🌳

Current target branch is `megaease:main`, if I should make some updates, please let me know. ❤️

File: doc/cookbook/api-aggregation.md [hunk 1]
```diff
@@ -183,8 +183,7 @@ flow:
   namespace: demo3
 - filter: proxy-demo1
   namespace: demo1
-  jumpIf:
-    serverError: "proxy-demo2"
+  jumpIf: { serverError: proxy-demo2 }
```
Comment (localvar) [inline doc/cookbook/api-aggregation.md:186]: why change this line? I think it is correct both before and after the change.

Comment (grootpiano) [reply doc/cookbook/api-aggregation.md:186]: > why change this line? I think it is correct both before and after the change.

That's right, both are correct. Just want to keep the same code style with other cookbook examples
e.g. in [Pipeline](https://github.com/megaease/easegress/blob/main/doc/cookbook/pipeline.md#built-in-filter-end): 
```yaml
- filter: validator
  jumpIf: { invalid: buildFailureResponse }
```
If there's no necessary, I can recover it.

Comment (localvar) [reply doc/cookbook/api-aggregation.md:186]: ok, that's fine.

**Review by localvar — APPROVED** [review]

**Review by haoel — APPROVED** [review]

**Review by suchen-sci — APPROVED** [review]

**Pull request merged**